app/models/computer_game.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class ComputerGame < ApplicationRecord
+  include ActiveModel::Validations
+
+  belongs_to :user
+  has_one :computer, dependent: :destroy
+  has_many :games, dependent: :destroy
+
+  validates_with ComputerGameValidator
+  validate :computer_is_used, on: :create
+
+  private
+
+  def computer_is_used
+    errors.add(:base, 'Computer is already used.') if ComputerGame.where(computer: computer).any?
```

Comment:
why not I18n? :) 

what will be when your client will want to go to for example german market?